### DISCUSSION TITLE
I can't display the view of the '<p-columnFilter>' component  and the view of the '<p-multiSelect>' component  simultaneously.

### DISCUSSION THREAD

**Discussion by yjseijas** [primefaces/community#2910] [PrimeNG]
The  p-columnFilter component shows a couple of combo boxes with the options 'Match All, Match Any ' and 'Starts with, Contains, etc...' . 
The p-multiSelect  component shows a list with the items of the respective column.
My question: is there a way to display both types of filters simultaneously?

By the way: I am working with Angular 18. Thanks!

**Comment by qwadrox** [comment]
@yjseijas Hi!

There is no way out of the box but you can build your custom solution relying on the templating. 
There is an example in the docs that is almost the same functionality you described: [The status column in the advanced example](https://primeng.org/table#filter-advanced)
In this example, the dropdown has a list with the items from the rows, and we can add multiple rules (replacing to have the multiselect). 

Now, you can place anything in the p-columnFilter through the templates and for instance, control the filtering from your component where you watch the Table with @ViewChild and call the `filter` method on actions. 

 You can try it and come back here if you can't build it, I'll help. Just create a sample, with the descriptions of your requirements on StackBlitz. 

I also did play around and placed a multiselect in the template but idk what you're trying to achieve exactly. It's working also: 
https://stackblitz.com/edit/ewjse1-ujwymk?file=src%2Fapp%2Ftable-filter-advanced-demo.html

```html
<div class="flex align-items-center">
  Status
  <p-columnFilter field="status" matchMode="contains" display="menu">
    <ng-template
      pTemplate="filter"
      let-value
      let-filter="filterCallback"
    >
      <p-multiSelect
        [ngModel]="value"
        [options]="statuses"
        placeholder="Any"
        (onChange)="filter($event.value)"
      >
        <ng-template let-option pTemplate="item">
          <p-tag
            [value]="option.value"
            [severity]="getSeverity(option.label)"
          />
        </ng-template>
      </p-multiSelect>
    </ng-template>
  </p-columnFilter>
</div>
```

**Reply by yjseijas** [reply]
Hi Boros. Thank you very much for your help! I will analyze everything you mentioned and if I have any questions I will let you know.
